src/main/java/org/jabref/gui/SaveOrderConfigDisplayView.java

```diff
@@ -20,17 +29,23 @@
     @FXML private CheckBox savePriDesc;
     @FXML private CheckBox saveSecDesc;
     @FXML private CheckBox saveTerDesc;
+    @Inject private PreferencesService preferencesService;
+
     private SaveOrderConfigDisplayViewModel viewModel;
-    private final MetaData metadata;
 
-    public SaveOrderConfigDisplayView(MetaData metadata) {
-        this.metadata = metadata;
+    public SaveOrderConfigDisplayView(SaveOrderConfig config) {
+        this.config = config;
+
+        ViewLoader.view(this)
+                  .root(this)
+                  .controller(this)
```

Comment:
@tobiasdiez  I finally managed it to load this thing as custom component. The only problem I have is that the `initialize()` method is not executed, although I did specify the controller.
https://docs.oracle.com/javafx/2/fxml_get_started/custom_control.htm

You can test my branch by using the Preferences -> Export Sorting
https://docs.oracle.com/javafx/2/fxml_get_started/custom_control.htm

Reply:
You need to add the `fx:controller` argument in the fxml (and the `controller`-call is already part of `view`). But I'm not sure if this really works. In either case, you can always call the initalize at the end of the constructor yourself.